RichTextArea does not properly show up in SceneBuilder

On the first import it says that there is a component:
![image](https://github.com/gluonhq/rich-text-area/assets/36427968/2eb97fc3-69d7-4c52-a2cd-43a1460273d5)

Yet there is nothing on the left hand side where all of the components are supposed to be.

Trying to delete and reimport leads to this screen:
![image](https://github.com/gluonhq/rich-text-area/assets/36427968/ca2a20ce-8763-4b88-a7d6-55f2df3a2d07)

I also tried to manually import the jar, but it leads to the screen above.

I would be nice to fix this so that SceneBuilder is usable with the jar. I also tried the gluon map package, and that works properly.

There is also this report in the Jar analysis section:

```Exception for: com/gluonhq/richtextarea/RichTextArea.class
java.io.IOException: java.lang.ExceptionInInitializerError
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.instantiateWithFXMLLoader(ExplorerBase.java:60)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.exploreEntry(ExplorerBase.java:111)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.exploreEntry(JarExplorer.java:83)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.explore(JarExplorer.java:65)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.exploreAndUpdateLibrary(LibraryFolderWatcher.java:381)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.runDiscovery(LibraryFolderWatcher.java:155)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.run(LibraryFolderWatcher.java:100)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ExceptionInInitializerError
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(Unknown Source)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$InstanceDeclarationElement.constructValue(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.load(Unknown Source)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.instantiateWithFXMLLoader(ExplorerBase.java:56)
	... 7 more
Caused by: java.lang.IllegalArgumentException: DataFormat 'text/rich-text-area' already exists.
	at javafx.graphics@21/javafx.scene.input.DataFormat.<init>(Unknown Source)
	at com.gluonhq.richtextarea.RichTextArea.<clinit>(RichTextArea.java:78)
	... 21 more


Exception for: com/gluonhq/richtextarea/ParagraphTile.class
javafx.fxml.LoadException: 
unknown path:2

	at javafx.fxml@21/javafx.fxml.FXMLLoader.constructLoadException(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$InstanceDeclarationElement.constructValue(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.load(Unknown Source)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.instantiateWithFXMLLoader(ExplorerBase.java:56)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.exploreEntry(ExplorerBase.java:111)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.exploreEntry(JarExplorer.java:83)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.explore(JarExplorer.java:65)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.exploreAndUpdateLibrary(LibraryFolderWatcher.java:381)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.runDiscovery(LibraryFolderWatcher.java:155)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.run(LibraryFolderWatcher.java:100)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodException: com.gluonhq.richtextarea.ParagraphTile.<init>()
	at java.base/java.lang.Class.getConstructor0(Unknown Source)
	at java.base/java.lang.Class.getDeclaredConstructor(Unknown Source)
	... 13 more


Exception for: com/gluonhq/richtextarea/BackgroundColorPath.class
javafx.fxml.LoadException: 
unknown path:2

	at javafx.fxml@21/javafx.fxml.FXMLLoader.constructLoadException(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$InstanceDeclarationElement.constructValue(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.load(Unknown Source)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.instantiateWithFXMLLoader(ExplorerBase.java:56)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.exploreEntry(ExplorerBase.java:111)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.exploreEntry(JarExplorer.java:83)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.explore(JarExplorer.java:65)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.exploreAndUpdateLibrary(LibraryFolderWatcher.java:381)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.runDiscovery(LibraryFolderWatcher.java:155)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.run(LibraryFolderWatcher.java:100)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodException: com.gluonhq.richtextarea.BackgroundColorPath.<init>()
	at java.base/java.lang.Class.getConstructor0(Unknown Source)
	at java.base/java.lang.Class.getDeclaredConstructor(Unknown Source)
	... 13 more


Exception for: com/gluonhq/richtextarea/RichListCell.class
javafx.fxml.LoadException: 
unknown path:2

	at javafx.fxml@21/javafx.fxml.FXMLLoader.constructLoadException(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$InstanceDeclarationElement.constructValue(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.processStartElement(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(Unknown Source)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.load(Unknown Source)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.instantiateWithFXMLLoader(ExplorerBase.java:56)
	at com.oracle.javafx.scenebuilder.kit.library.util.ExplorerBase.exploreEntry(ExplorerBase.java:111)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.exploreEntry(JarExplorer.java:83)
	at com.oracle.javafx.scenebuilder.kit.library.util.JarExplorer.explore(JarExplorer.java:65)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.exploreAndUpdateLibrary(LibraryFolderWatcher.java:381)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.runDiscovery(LibraryFolderWatcher.java:155)
	at com.oracle.javafx.scenebuilder.kit.library.user.LibraryFolderWatcher.run(LibraryFolderWatcher.java:100)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodException: com.gluonhq.richtextarea.RichListCell.<init>()
	at java.base/java.lang.Class.getConstructor0(Unknown Source)
	at java.base/java.lang.Class.getDeclaredConstructor(Unknown Source)
	... 13 more
```

--- a/rta/src/main/java/com/gluonhq/richtextarea/ParagraphTile.java
+++ b/rta/src/main/java/com/gluonhq/richtextarea/ParagraphTile.java
@@ -80,7 +80,7 @@ import java.util.stream.Stream;
 
 import static com.gluonhq.richtextarea.model.TableDecoration.TABLE_SEPARATOR;
 
-public class ParagraphTile extends HBox {
+class ParagraphTile extends HBox {
 
     private static final double INDENT_PADDING = 20.0;
 

--- a/rta/src/main/java/com/gluonhq/richtextarea/RichTextArea.java
+++ b/rta/src/main/java/com/gluonhq/richtextarea/RichTextArea.java
@@ -76,7 +76,11 @@ import java.util.function.Function;
 public class RichTextArea extends Control {
 
     public static final String STYLE_CLASS = "rich-text-area";
-    public static final DataFormat RTA_DATA_FORMAT = new DataFormat("text/rich-text-area");
+    public static final DataFormat RTA_DATA_FORMAT;
+    static {
+        DataFormat dataFormat = DataFormat.lookupMimeType("text/rich-text-area");
+        RTA_DATA_FORMAT = dataFormat == null ? new DataFormat("text/rich-text-area") : dataFormat;
+    }
     private static final PseudoClass PSEUDO_CLASS_READONLY = PseudoClass.getPseudoClass("readonly");
 
     private final ActionFactory actionFactory = new ActionFactory(this);

